Library/Homebrew/cmd/update.sh

```diff
@@ -354,10 +354,10 @@ EOS
   if [[ ! -w "$HOMEBREW_REPOSITORY" ]]
   then
     odie <<EOS
-$HOMEBREW_REPOSITORY is not writable. You should change the
-ownership and permissions of $HOMEBREW_REPOSITORY back to your
+$HOMEBREW_REPOSITORY folders are not writable. You should change the
+ownership and permissions of $HOMEBREW_REPOSITORY folder back to your
```

Comment:
```suggestion
ownership and permissions of $HOMEBREW_REPOSITORY folders back to your
```

Reply:
“subdirectories”

Reply:
`HOMEBREW_REPOSITORY` is `/usr/local/Homebrew` since Homebrew 1.0. I'm very confused as to how you'd not be able to `chown` that?

Reply:
Then I found the wrong message. I thought this was `/usr/local`

Reply:
I'd love to be proved wrong but I'm pretty sure the only way you'll see the old `/usr/local` message is if you're only a really old version of Homebrew.